packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.9.2
```

Comment:
Nit: 0.9.1+1 per the discussion of pre-1.0 versions [here](https://dart.dev/tools/pub/versioning#semantic-versions).